examples/connector/golang/go.sum

```diff
@@ -1,161 +1,21 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
This file was changed when I ran the "go mod tidy" command as suggested by the Go compiler